src/h5web/visualizations/shared/utils.ts

```diff
@@ -10,11 +11,17 @@ import {
   ScaleType,
 } from './models';
 
+const TICK_FORMAT = format('0');
+
 export const adaptedNumTicks = scaleLinear()
   .domain([300, 900])
   .rangeRound([3, 10])
   .clamp(true);
 
+const adaptedLogTicksThreshold = scaleLinear()
+  .domain([300, 500])
+  .range([0.8, 1.4]);
```

Comment:
How did you determine the threshold values ?

Reply:
Same way we determined the values for `adaptedNumTicks`: I picked a height and found the threshold that worked for it, then picked another height and did the same. Granted, the thresholds may not work as well for the horizontal axis... but I figured we'd deal with this in due time. One problem at a time! 😄